### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by joemccla**
Based on user feedback, it was unclear that the policy here could be set more granularly.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Watch "Meeting Recording" for more information.
    Set-CsTeamsMeetingPolicy -Identity Global -RecordingStorageMode "OneDriveForBusiness"
 ```
 
+This example shows the policy being set globally, but the policy can be set [more granularly just like other Teams policies](https://docs.microsoft.com/en-us/MicrosoftTeams/assign-policies).
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:89]:
```suggestion
This example shows the policy being set globally, but the policy can be set [more granularly just like other Teams policies](https://docs.microsoft.com/MicrosoftTeams/assign-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please remove the locale from the link.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cichur** [comment]
Hi @JohanFreelancer9  I've added another Note to this article that describes the nuances of setting a global policy versus setting a policy at the Teams policy level. It will be live this afternoon.

**Pull request closed without merging**